aclogview/CM_Combat.cs

```diff
@@ -72,7 +73,15 @@ public class TargetedMeleeAttack : Message {
         public override void contributeToTreeView(TreeView treeView) {
             TreeNode rootNode = new TreeNode(this.GetType().Name);
             rootNode.Expand();
-            rootNode.Nodes.Add("i_targetID = " + i_targetID);
+            if (Globals.UseHex)
+            {
+                rootNode.Nodes.Add("i_targetID = " + "0x" + i_targetID.ToString("X"));
+            }
+            else
+            {
+                rootNode.Nodes.Add("i_targetID = " + this.i_targetID);
+            }
```

Comment:
I'm thinking these blocks should be done via a separate function, since this is turning a single line into 8 lines and causes every treeview contributor to reach into globals. What about something like:

rootNode.Nodes.Add("i_targetID = " + formatIDForDisplay(this.i_targetID));

Where formatIDForDisplay would internally check UseHex to either just return the input as a string or "0x" + input.ToString("X"); if UseHex is enabled.